lib/entities/scheduled-action.ts

```diff
@@ -62,8 +73,26 @@ export type ScheduledActionProps = {
      */
     timezone?: string
   }
+  /**
+   * The Contentful-style error that occurred during execution if sys.status is failed
+   *
+   * @example
+   * {
+   *   sys: {
+   *     type: 'Error',
+   *     id: 'InvalidEntry'
+   *   },
+   *   message: 'Entry is invalid',
+   *   details: {
+   *     errors: [...]
+   *   }
+   * }
+   */
+  error?: ScheduledActionFailedError
 }
 
+export type CreateUpdateScheduledActionProps = Omit<ScheduledActionProps, 'sys' | 'error'>
```

Comment:
maybe we can go the opposite route as well: instead of `Omit` we could `Pick` to explicitly say which props we want